bepasty/utils/permissions.py

```diff
@@ -54,7 +54,7 @@ def get_permissions():
 def get_permission_icons():
     return [
         (permission, permission_icons[permission])
-        for permission in get_permissions()
+        for permission in get_permissions() if permission
```

Comment:
hmm, that should be rather the empty set (not  set with 1 empty-string member).
but ok, that bug is not in your stuff.